### PR TITLE
Add health insurances from Liechtenstein

### PR DESCRIPTION
**Pull request by Double-A-92**

### CODE REVIEW

File: companies/fkb.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        "li"
+    ],
+    "categories": [
+        "health"
+    ],
+    "name": "FKB – Die liechtensteinische Gesundheitskasse",
+    "address": "Gagoz 75\nPostfach 363\n9496 Balzers\nLiechtenstein",
+    "phone": "+423 388 19 90",
+    "fax": "+423 388 19 91",
+    "email": "info@fkb.li",
```
Comment (mal-tee) [inline companies/fkb.json:13]: > oder unter info@fkb.li - Betreff beginnend mit „[Datenschutzbeauftragte]:“ - erreichbar. 

You could add this as a comment. @baltpeter Do we want to support something like this in the long run? :D

File: companies/swica.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    "name": "SWICA Krankenversicherung AG",
+    "address": "Römerstrasse 38\n8401 Winterthur\nSchweiz",
+    "phone": "+41 52 244 22 33",
+    "fax": "+41 52 244 22 90",
+    "email": "datenschutz@swica.ch",
+    "web": "https://www.swica.ch/",
+    "sources": [
+        "https://www.swica.ch/de/ueber-swica/portrait/ueberblick/datenschutz",
+        "https://www.swica.ch/de/ueber-swica/portrait/ueberblick/datenschutz/datenschutzerklaerung",
+        "https://www4.swica.ch/p/826_d_Datenschutzerklaerung_Liechtenstein.pdf",
```
Comment (mal-tee) [inline companies/swica.json:19]: What did I miss, what is this source covering? :D

File: companies/fkb.json (hunk 1 above)
Comment (baltpeter) [reply companies/fkb.json:13]: No. And I don't think it's necessary to add that as a comment either. Our subjects should already be more than clear enough. And they aren't legally allowed to force you to add that to the subject.

Comment (mal-tee) [reply companies/fkb.json:13]: Ok!

**Pull request merged**